### PR TITLE
clean up _insertFeature

### LINKED ISSUE #38: _insertFeature cleanup
I noticed two things that are not being used in _insertFeature
- There is a parameter for defining id but it is never used: https://github.com/koopjs/koop-pgcache/blob/master/index.js#L646
- The crs is being set on the geometry after the feature is stringified and after the geohash is created so I don't think it would have any effect: https://github.com/koopjs/koop-pgcache/blob/master/index.js#L654

I have a branch that addresses these issues and can make a pull request but wanted to get feedback first as I am new to this repo.
https://github.com/koopjs/koop-pgcache/compare/master...kneemer:insertfeature-cleanup

### PR DESCRIPTION
**Pull request by kneemer**
Remove unused code.  Resolves #38

### CODE REVIEW

**Comment by ungoldman** [comment]
@kneemer mind adding a line to the change log? otherwise looks great :grin:

**Comment by kneemer** [comment]
@ngoldman is this what you had in mind?

**Comment by kneemer** [comment]
Well, looks like that change triggered the random Travis failure #35.   Unless he doesn't like CHANGELOGs :)

**Comment by ungoldman** [comment]
Also a note about removing CRS as that's a little more major

**Comment by kneemer** [comment]
Well, it wasn't actually using it.

**Comment by kneemer** [comment]
It was getting set after stringifying and geohashing the feature and not used in the insert query at all.

**Comment by ungoldman** [comment]
@kneemer was the crs getting attached to the geojson object returned to the client?

File: index.js [hunk 1]
```diff
@@ -643,15 +643,13 @@ module.exports = {
...
-   * @param {string} i - index value to use an id
    * @private
    */
-  _insertFeature: function (table, feature, i) {
+  _insertFeature: function (table, feature) {
     var featurestring = JSON.stringify(feature).replace(/'/g, '')
 
     if (feature.geometry && feature.geometry.coordinates && feature.geometry.coordinates.length) {
       var geohash = this.createGeohash(feature, this.geohashPrecision)
-      feature.geometry.crs = {'type': 'name', 'properties': {'name': 'EPSG:4326'}}
```
Comment (ungoldman) [inline index.js:605]: so `feature` isn't even used after this point, right @kneemer? this is just a weird artifact?

**Comment by ungoldman** [comment]
@kneemer sorry merged another pull request, can you rebase? other than that it looks good time, thanks!

File: index.js (hunk 1 above)
Comment (kneemer) [inline index.js:605]: correct

**Comment by kneemer** [comment]
@ngoldman rebase complete

**Comment by ungoldman** [comment]
:+1:

**Pull request merged**